src/component-library/Input/Input.style.tsx

```diff
@@ -13,14 +17,17 @@ const BaseInput = styled.input`
   font: inherit;
   letter-spacing: inherit;
   background: none;
-  color: currentcolor;
+  color: ${(props) => {
```

Comment:
I think I'd find a ternary a little cleaner here - we're only checking one prop. We haven't discussed a convention around this, but for inline checking I'd lean more towards using a ternary than if statements.

Reply:
Which I notice is what you've done below :)